stl/inc/vector

```diff
@@ -1039,6 +1039,12 @@ public:
         pointer& _Myfirst = _My_data._Myfirst;
         pointer& _Mylast  = _My_data._Mylast;
 
+#if _ITERATOR_DEBUG_LEVEL == 2
+        const_pointer _Valptr = addressof(_Val);

```

Comment:
I am unsure whether this works with "fancy pointers". `addressof` returns a raw pointer `const _Ty*` but the `const_pointer` typedef can be a "fancy pointer" type provided by the allocator. (We have functions to convert fancy pointers into raw, see `_Unfancy`.)

Reply:
You are right. One of the failed test was about fancy pointers.  _Unfancy did the work. Thank you for your hint and support.